libcontainer/init_linux.go

```diff
@@ -179,6 +179,16 @@ func startInitialization() (retErr error) {
 		defer consoleSocket.Close()
 	}
 
+	var pidfdSocket *os.File
+	if envConsole := os.Getenv("_LIBCONTAINER_PIDFD"); envConsole != "" {
```

Comment:
1. Copy from the prev lines but doesn’t change the var name.
2. _LIBCONTAINER_PIDFD -> _LIBCONTAINER_PIDFD_SOCK

Reply:
And could you please add an integration test?

Reply:
> Copy from the prev lines but doesn’t change the var name.

You mean that `envConsole` ?  The `envConsole` shadows the previous one.

> And could you please add an integration test?

Sure. I will add the integration test when the maintainers accept this feature request 😂

Reply:
> You mean that `envConsole` ? The `envConsole` shadows the previous one.

Yes, ‘envConsole’ and ‘console’.
I think you copied them from other place, but you should change the variables name even they aren’t in the same scope.

Reply:
Updated